config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/studios", to: "studios#index"
+
+  get "/movies/:id", to: "movies#show"
+  patch "/movies/:id", to: "movies#update"
```

Comment:
Consider the more RESTful option: `post "/movies/:id/actors", to: "movie_actors#create"`